core/controllers/collection_viewer.py

```diff
@@ -46,7 +47,7 @@ def test_can_play(self, collection_id, **kwargs):
 class CollectionPage(base.BaseHandler):
     """Page describing a single collection."""
 
-    @require_collection_playable
+    @action_decorators.play_collection
```

Comment:
What do you think about @acl_decorators.can_play_collection? (I.e. insert the word "can")